monitor/api/PythonVirtualMachine.cpp

```diff
@@ -111,7 +111,7 @@ bool PythonVirtualMachine::Initialize()
   if (embedded)
   {
     // try and load the python core files.
-    MYODD_STRING exe_dir = myodd::files::GetAppPath(true);
```

Comment:
The `GetAppPath( ... )` should return a const& and the consumers should use `const auto&`